### PR TITLE
add: Tween Leaf for Behaviour Tree

### PR DESCRIPTION
**Pull request by ThePat02**

### CODE REVIEW

**Comment by squk** [comment]
Very nice!

File: addons/behaviour_toolkit/behaviour_tree/leaves/leaf_tween.gd [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+var tween : Tween
+
+
+func tick(actor: Node, _blackboard: Blackboard) -> Status:
+	# Initialize tween, if not already initialized
+	_init_tween(actor)
+
+	if tween.is_running():
+		return Status.RUNNING
+	return Status.SUCCESS
```
Comment (squk) [inline addons/behaviour_toolkit/behaviour_tree/leaves/leaf_tween.gd:67]: nit: add a call to `tween.kill()` before exiting?

**Comment by squk** [comment]
An exported bool for Tween's `as_relative` would be nice to have. Thanks for this!

**Comment by ThePat02** [comment]
> An exported bool for Tween's `as_relative` would be nice to have. Thanks for this!

What do you exactly mean by `as_relative`?

**Comment by squk** [comment]
https://docs.godotengine.org/en/stable/classes/class_propertytweener.html#class-propertytweener

`PropertyTweener.as_relative( )`

**Comment by ThePat02** [comment]
@squk Added your suggestions! Can you please verify the tween actually works in your project?

**Comment by squk** [comment]
`Invalid call. Nonexistent function 'set_loop' in base 'Tween'.`

`set_loop` is actually a property of `PropertyTweener`, like `as_relative()`.

**Comment by ThePat02** [comment]
> Invalid call. Nonexistent function 'set_loop' in base 'Tween'.

Fixed the typo!

> For my own curiosity, what program do you use to make these SVG files?

I use Inkscape! Very easy to throw something together. Just mix and match the Godot Editor Icons, other CC0 icons and your own shapes and you are good to go!

**Pull request merged**